### DISCUSSION TITLE
Reuse info.plist and other files

### DISCUSSION THREAD

**Discussion by PavloLukianets** [dotnet/maui#16779] [Q&A]
Currently i am trying to link the existing info.plist and other files such as Entitlements, GoogleService-Info.plist to reuse them in our old xamarin project and new Maui project. Is it possible?
I am expecting something like this to work
```csharp
<ItemGroup>
        <CodesignEntitlements Include="..\..\iOS\PF.iOS\Entitlements.plist">
            <Link>Platforms\iOS\Entitlements.plist</Link>
        </CodesignEntitlements>
        <Content Include="..\..\iOS\PF.iOS\Info.plist">
            <Link>Platforms\iOS\Info.plist</Link>
        </Content>
        <Content Include="..\..\iOS\PF.iOS\Resources\en.lproj\InfoPlist.strings">
            <Link>Platforms\iOS\Resources\en.lproj\InfoPlist.strings</Link>
        </Content>
        <Content Include="..\..\iOS\PF.iOS\Resources\es.lproj\InfoPlist.strings">
            <Link>Platforms\iOS\Resources\es.lproj\InfoPlist.strings</Link>
        </Content>
    </ItemGroup>
```

**Comment by baaaaif** [comment]
I tried something similar, but File-Linking didn't work.
My solution for now:
Copy the files with the Pre-build event
something like

copy /Y "..\..\AppLabels\$(_AppLabelFolderName)\Info.plist" "$(ProjectDir)Platforms\iOS\Info.plist"
copy /Y "..\..\AppLabels\$(_AppLabelFolderName)\Entitlements.plist" "$(ProjectDir)Entitlements.plist"
